admin_guide/out_of_resource_handling.adoc

```diff
@@ -270,21 +496,37 @@ xref:../admin_guide/overcommit.adoc#qos-classes[quality of service], and, among
 those with the same quality of service, by the consumption of the starved
 compute resource relative to the pod's scheduling request.
 
-* `BestEffort`: pods that consume the most of the starved resource are failed
+Each QOS level has an OOM score, which the Linux out-of-memory tool (OOM killer) uses to determine which pods to kill. See xref:out-of-resource-node-out-of-resource-and-out-of-memory[Understanding Quality of Service and Out of Memory Killer] below.
+
+The following table lists each QOS level and the associated OOM score.
+
+.Quality of Service Levels
+[cols="3a,8a",options="header"]
+|===
+
+| Quality of Service | Description
+
+|`Guaranteed`
+| Pods that consume the highest amount of the starved resource relative to
+their request are failed first. If no pod has exceeded its request, the strategy
+targets the largest consumer of the starved resource.
+
+|`BestEffort`
+| Pods that consume the highest amount of the starved resource are failed
 first.
-* `Burstable`: pods that consume the most of the starved resource relative to their
+
+|`Burstable`
```

Comment:
I'd move Burstable above BestEffort in the table to order from least-to-most likely to be evicted i.e. highest-to-lowest QoS.